style.css

```diff
@@ -17,6 +17,14 @@ h1, h2{
     font-weight:normal;
 }
 
+h1{

```

Comment:
In my CSS file, I began working on some font changes for a potential font scheme for the next task.